src/App.js

```diff
@@ -7,20 +7,23 @@ import SignUp from "./frontend/pages/Auth/SignUp/SignUp";
 import MockAPI from "./frontend/mock/MockAPI";
 import UserProfile from "./frontend/pages/UserProfile/UserProfile";
 import { ToastContainer } from "react-toastify";
+import NewNote from "./frontend/pages/NewNote/NewNote";
+import Notes from "./frontend/pages/Notes/Notes";
```

Comment:
use Barrel Imports

Reply:
Thanks for the review,  I will do the changes while refactoring the code